samples/grids/grid/binding-remote-data/NwindData.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+public class NwindDataItem
```

Comment:
rename class to ProductDataItem and move it to Services\ProductDataService.cs file because NwindData.cs witout actual data values would conflict with samples that use full NwindData.cs file